src/js/lib/api.js

```diff
@@ -0,0 +1,17 @@
+import { buildSearchQuery } from "./search"
+
+export const search = async params => {
+  const body = buildSearchQuery(params)
+
+  const response = await fetch("http://localhost:8063/api/v0/search/", {
```

Comment:
Looks like this is hardcoded, do you have a plan for making this configurable?

Reply:
yes - the code isn't quite final yet so I haven't cleaned up all these things yet